package.json

```
@@ -17,6 +17,7 @@
         "i18n": "nx xi18n --output-path src/locale",
         "*** Workflow ***": "",
         "test": "nx test --browsers=ChromeHeadless --code-coverage=true --",
+        "many": "npm run-script env -- nx run-many --parallel --maxParallel=5",
```

`npm run-script env`
зачем?
`maxParallel=5` кажется, что это too mach для одного агента.

```suggestion
        "many": "nx run-many --parallel --maxParallel=3",
```

---

what do you think about move this parameter to environments?

`on ci`

```
$ npm run many -- --maxParallel=${{ NX_PARALLEL_COUNT }}
```

or another name for `NX_PARALLEL_COUNT`?

---

Maybe `NX_MAX_PARALLEL`?